internal/commands/iam/groups/iam/updater.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package iam
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-resource-manager/stable/2019-12-10/client/resource_service"
+	"github.com/hashicorp/hcp-sdk-go/clients/cloud-resource-manager/stable/2019-12-10/models"
+	"github.com/hashicorp/hcp/internal/pkg/api/iampolicy"
+)
+
+// iamUpdater meets the iampolicy.ResourceUpdater interface. It is used to
+// manage IAM bindings.
+type iamUpdater struct {
+	groupID string
```

Comment:
Can't you take the resource_name: https://github.com/hashicorp/hcp-sdk-go/blob/main/clients/cloud-resource-manager/stable/2019-12-10/models/hashicorp_cloud_resourcemanager_resource_set_iam_policy_request.go#L29-L30?

This can save an API request to convert the name -> ID.